docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+[[timeline-template-api-update]]
+=== Update Timeline or Timeline template
+
+Updates an existing Timeline or Timeline template.
+
+Use the `timeline` object's <<timeline-object-typeField, `timelineType`>> field
+to specify whether a Timeline or a Timeline template is updated, where:
+
+* `"timelineType": "default"` updates an existing Timeline
+* `"timelineType": "template"` updates an existing Timeline template
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/timeline`
+
+==== Request body
+
+A JSON object defining the Timeline or Timeline template query and time filter.
+
+NOTE: For detailed information about the Timeline object schema and its
+corresponding UI components, see <<timeline-object-schema>>.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`timeline` |<<timeline-object-schema, timeline>> a|The timeline object, which
+defines the search criteria and time range.
+
+When you are updating a Timeline template, provide these fields so you can
+easily <<timeline-api-import, import template>> updates:
+
+* `templateTimelineId`: Unique identifier (UUID, for Timeline templates only). This must match the `templateTimelineId` of an existing Timeline template that you're updating.
```

Comment:
What are your thoughts on combining these sentences and making the description more actionable? Maybe something like this: 

```suggestion
* `templateTimelineId`: Use the unique identifier (UUID) of the Timeline template that you're updating. 
```